modin/pandas/test/test_dataframe.py

```diff
@@ -3328,279 +3136,144 @@ def test_describe(self, data):
-            )
-        except Exception as e:
-            with pytest.raises(type(e)):
-                modin_df.T.mean(axis=axis, skipna=skipna, numeric_only=numeric_only)
-        else:
-            modin_result = modin_df.T.mean(
-                axis=axis, skipna=skipna, numeric_only=numeric_only
-            )
-            df_equals(modin_result, pandas_result)
+        modin_df, pandas_df = pd.DataFrame(data), pandas.DataFrame(data)
+        assert modin_df.last_valid_index() == pandas_df.last_valid_index()
 
-    @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
     @pytest.mark.parametrize(
         "index", bool_arg_values, ids=arg_keys("index", bool_arg_keys)
     )
+    @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
     def test_memory_usage(self, data, index):
-        modin_df = pd.DataFrame(data)
-        pandas_df = pandas.DataFrame(data)  # noqa F841
-
-        modin_result = modin_df.memory_usage(index=index)
-        pandas_result = pandas_df.memory_usage(index=index)
-        df_equals(modin_result, pandas_result)
+        eval_general(*create_test_dfs(data), lambda df: df.memory_usage(index=index))
 
-    @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-    @pytest.mark.parametrize("axis", axis_values, ids=axis_keys)
-    @pytest.mark.parametrize(
-        "skipna", bool_arg_values, ids=arg_keys("skipna", bool_arg_keys)
-    )
+    @pytest.mark.parametrize("method", ["min", "max", "mean"])
+    @pytest.mark.parametrize("is_transposed", [False, True])
     @pytest.mark.parametrize(
         "numeric_only", bool_arg_values, ids=arg_keys("numeric_only", bool_arg_keys)
     )
-    def test_min(self, data, axis, skipna, numeric_only):
-        modin_df = pd.DataFrame(data)
-        pandas_df = pandas.DataFrame(data)
-
-        try:
-            pandas_result = pandas_df.min(
-                axis=axis, skipna=skipna, numeric_only=numeric_only
-            )
-        except Exception:
-            with pytest.raises(TypeError):
-                modin_df.min(axis=axis, skipna=skipna, numeric_only=numeric_only)
-        else:
-            modin_result = modin_df.min(
-                axis=axis, skipna=skipna, numeric_only=numeric_only
-            )
-            df_equals(modin_result, pandas_result)
-
-        try:
-            pandas_result = pandas_df.T.min(
-                axis=axis, skipna=skipna, numeric_only=numeric_only
-            )
-        except Exception:
-            with pytest.raises(TypeError):
-                modin_df.T.min(axis=axis, skipna=skipna, numeric_only=numeric_only)
-        else:
-            modin_result = modin_df.T.min(
-                axis=axis, skipna=skipna, numeric_only=numeric_only
-            )
-            df_equals(modin_result, pandas_result)
-
-    @pytest.mark.skipif(
-        os.name == "nt",
-        reason="Windows has a memory issue for large numbers on this test",
-    )
```

Comment:
Is this issue was fixed?

Reply:
I am not sure, but this issue in current state is not reproducible.

Reply:
Returned this mark, since my goal is not to fix the tests.